build.sbt

```diff
@@ -184,6 +184,8 @@ lazy val commonSettings = Seq(
 
   homepage := Some(url("https://github.com/druid-io/tranquility")),
 
+  resolvers += "sigar" at "https://repository.jboss.org/nexus/content/repositories/thirdparty-uploads/",
```

Comment:
I'll remove it with new `scala-util` version. In one of previous versions of `server-metrics` library we removed `optional` for sigar. Now we made sigar optional again.